ci/pyuvdata_min_deps_tests.yml

```diff
@@ -3,9 +3,8 @@ channels:
   - conda-forge
 dependencies:
   - astropy>=5.0.4
-  - numpy>=1.19.*
-  - scipy>=1.3
```

Comment:
I don't think we want to drop scipy from this yaml. Probably happened in a rebase.

Reply:
ugh. Will fix.